app/models/custom_attribute.rb

```diff
@@ -2,7 +2,17 @@ class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   serialize :serialized_value
 
+  def value=(value)
```

Comment:
Outside perspective: this seems odd to me - specifically having a field called "serialized_value". Why is it not simply "value"? IMO serializing and deserializing is an implementation detail of the model which should not leak into its public API.

How is it serialized? There is good support in Postgres for querying JSON, for example